CODEOWNERS

```diff
@@ -1 +1 @@
-*	@brianloss @ivakegg @drewfarris @keith-ratcliffe @cawaring @tomnelson @jwomeara @billoley @matthpeterson @bbux-atg @alerman @hlgp @jschmidt10 @apmoriarty
+*	@brianloss @ivakegg @drewfarris @keith-ratcliffe @cawaring @tomnelson @jwomeara @billoley @matthpeterson @ejrgilbert @alerman @hlgp @jschmidt10 @apmoriarty
```

Comment:
Might I suggest that we stop maintaining this list and finally delete it like we talked about? 

```suggestion
```